.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Integration tests
+on:
+  [push, pull_request]
+
+jobs:
+  initialize-repository:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 3
```

Comment:
I don't remember why do we need `fetch-depth: 3`. Why not 1? Am I missing something?

Reply:
@aquilamacedo any comment here?

Reply:
Upon closer examination of this option, I now think that a fetch depth of 1 would be more appropriate. There is no necessity to delve into the historical data for these particular operations.

Reply:
@aquilamacedo please update your PR.